pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/Housekeeping.java

```diff
@@ -264,7 +293,7 @@ public static void start(String[] args, SendGridSupplier sendGridSupplier) {
                 Subscription subscription = pubsubConnectionManager
                         .createSubscriptionIfNotExists(projectSubscriptionName, projectTopicName);
                 // in the real world, listen differently
-                setupMessageReceiver(pubsubConnectionManager, projectSubscriptionName, cfg, sendGridSupplier);
+                setupMessageReceiver(pubsubConnectionManager, projectSubscriptionName, cfg, sendGridSupplier, proxy);
```

Comment:
It would be preferable to update the configuration to reflect the correct value of the proxy (`null` or the empty string, in this case) instead of adding an additional parameter.

Reply:
I will go this way, I have tried with empty  string and it did not work.
I hope with null it will work properly.

There is another point.

Application.conf is updated from vault on every configuration rendering.
and this happens almost on every build (at least very frequently).
That's why I prefer not to change anything in application.conf if this is possible